### ISSUE TITLE
Display flash message after submitting a new vacancy

### ISSUE THREAD

**Issue by davydovanton** [davydovanton/rubyjobs.dev#13]
## What

We want to display positive flash message after adding a new vacancy. Something like this:

```
Вакансия успешно отправлена на модерацию. В ближайшее время она появится на главной.
```

## Why

We want to make better user experience and flash message can help with dropping unnecessary uneasiness for anyone who submit a new vacancy.

## How
Useful links

* [Template example](https://github.com/ossboard-org/ossboard/blob/a121164eb8b88ceed4722dbdbcb3a46147354f39/apps/web/templates/tasks/index.html.slim#L2-L4)
* [Action example](https://github.com/ossboard-org/ossboard/blob/c28a61ab53ff810dda96e3e8297faab19dcfd382/apps/web/controllers/tasks/update.rb#L30)
* [How to test flash message](https://guides.hanamirb.org/actions/testing/#flash-messages)

**Comment by davydovanton** [comment]
closed by #19

**Comment by AlexWayfer** [comment]
Is it deployed? Seems like no.
